middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -2372,16 +2376,21 @@ let bind_code_and_sets_of_closures all_code sets_of_closures acc body =
       incr i;
       n
   in
-  let group_to_bound_consts, symbol_to_groups =
+  (* CR gbury: We assume that no code_ids are deleted later, but even if that
```

Comment:
Could you clarify that this comment is only meaningful for Classic mode, where the accumulated toplevel slot offsets are passed to `To_cmm` directly ?

Reply:
done